gateway/.s2i/bin/assemble

```diff
@@ -3,7 +3,9 @@ set -euo pipefail
 
 mkdir -p /tmp/.s2i
 
-if [[ -n "${GIT_BRANCH:-}" && "${GIT_BRANCH:-}" != master && -z "${GIT_TAG:-}" ]]; then
+PATCH_BRANCH=THREESCALE
+
+if [[ -n "${GIT_BRANCH:-}" && "${GIT_BRANCH:-}" != master && "${GIT_BRANCH:-}" != *"${PATCH_BRANCH}"* ]]; then
```

Comment:
I believe this was maybe included by mistake in this PR?

Reply:
well it's not a big deal but yeah cleaner to remove it I guess

Reply:
Done!